.lando.yml

```diff
@@ -53,35 +53,26 @@ services:
   #       - /dev/shm:/dev/shm
   #     command: /opt/bin/entry_point.sh
   # elasticsearch:
-  #   type: compose
-  #   meUser: elasticsearch
-  #   ssl: true
-  #   sslExpose: false
-  #   services:
-  #     image: blacktop/elasticsearch:7
-  #     command: /elastic-entrypoint.sh elasticsearch
-  #     ports:
-  #       - "9200:9200"
-  #       - "9300:9300"
-  #     volumes:
-  #       - ./.lando/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml
```

Comment:
This leaves `.lando/elasticsearch.yml` unused. Also, there needs to be an alternative to customize ES setup, see https://github.com/wunderio/drupal-project/pull/69#issuecomment-537369681.

Reply:
Ok, this should cover the customisation part: https://github.com/lando/lando/blob/master/examples/elasticsearch/.lando.yml#L14